omero/sysadmins/public.txt

```diff
@@ -0,0 +1,135 @@
+Public data in the repository
+=============================
+
+The OMERO.web framework allows raw data to be published using built-in tools
+or supplied through webservices to the external web resources. This feature is
+available only after granting access to the selected datasets and making them
```

Comment:
This kinda sounds like you need to grant access **before** you can get started with "Public data" feature (as if you should have done that already).
Maybe say "Selected datasets can be made available to a 'public user' using the standard OMERO permissions system".